modules/tinymce/src/plugins/quickbars/test/ts/browser/PredicateTest.ts

```diff
@@ -0,0 +1,69 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/quickbars/Plugin';
+
+import { pAssertToolbarNotVisible, pAssertToolbarVisible } from '../module/test/Utils';
+
+describe('browser.tinymce.plugins.quickbars.PredicateTest', () => {
```

Comment:
Should this test/file be called `InsertToolbarPredicateTest` or even `InsertToolbarTest` (to match `SelectionToolbarTest`)? As this sounds like it's generic, but this only tests the insert quickbar and nothing else.